decidim-core/lib/decidim/core/engine.rb

```diff
@@ -32,6 +32,7 @@
 require "premailer/rails"
 require "nokogiri"
 require "geocoder"
+require "letter_opener_web"
```

Comment:
Well.. I'm not sure where I can include this 😄 . It crashes if I try to include it on `decidim-dev/lib/dev/dev.rb`. It was there in the first place but I needed to move it since I made those changes. Any idea?